packages/compiler-sfc/src/style/cssVars.ts

```diff
@@ -27,12 +26,14 @@ export function genCssVarsFromList(
     .join(',\n  ')}\n}`
 }
 
+const cssVarNameEscapeSymbolsRE = /[ !"#$%&'()*+,./:;<=>?@[\\\]^`{|}~]/g
```

Comment:
If we have two rules, we should put them together so that if we need to change one rule in the future, we can ensure that the second one is not missed and leads to inconsistency.

Reply:
Makes sense! I moved both regexps to `scripts/utils.ts`